### DISCUSSION TITLE
重复：不看说明系列

### DISCUSSION THREAD

**Discussion by caserangle** [hooke007/mpv_PlayKit#485] [回收]
![image](https://github.com/user-attachments/assets/86f452cc-98ed-4193-a263-326118f93a7e)
必须等这个CMD窗口自行关闭，视频才会播放 AI补帧才会生效，且中间过程显卡占用会到80-90） 配置是7900X 4090，请问这是跑什么着色器缓存嘛，是必须要跑一次才能看网络流视频嘛，本地视频倒是没出现类似情况。

**Comment by hooke007** [comment]
去看说明，本来就有这个东西 https://github.com/hooke007/MPV_lazy/discussions/123
